src/index.js

```diff
@@ -12,6 +13,7 @@ const app = express()
 
 const server = new ApolloServer({
   ...schema,
+  context: { db },
```

Comment:
You're not using this context on your resolvers. Either remove it or fix the name of `db` to `models`.